components/_main.scss

```diff
@@ -14,6 +14,7 @@
 @import 'layout';
 @import 'link';
 @import 'navigation';
+@import 'notification';
```

Comment:
Tror dette blir et for gennerelt navn. Er mye som er en notification 🤔 
Forslag: notification-indicator, notification-badge, notification-dot

Reply:
Liker notification-dot! Den er mer beskrivende ja